pkg/kando/location_profile.go

```diff
@@ -0,0 +1,92 @@
+package kando
+
+import (
+	"context"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/kanisterio/kanister/pkg/kopia/snapshot"
+)
+
+type Profile struct {
+	Command   *cobra.Command
+	Arguments []string
```

Comment:
I would not make Command/Arguments fields of any datamover implementation.
Datamover should just hold logic of managing data - Pull/Push/Delete for now.
We should move the datamover implementation to separate pkg `pkg/datamover/`.
The command and arg parsing should be in kando pkg. The handler should create object of Datamover (profile/kopia based of arg) and call data management methods (pull/push).

Reply:
Will work on it